packages/splitter/src/useSplitter.ts

```diff
@@ -146,7 +146,11 @@ export function useSplitter({
     [orientation, setBoundedDimension, rtl, environment]
   );
 
-  const onMouseMove = composeEventHandlers(props.onMouseMove, onSplitterMouseMove);
+  // Any events that are registered globally to the DOM need to conserve their reference for removal to prevent listener leaks
+  const onMouseMove = useMemo(
+    () => composeEventHandlers(props.onMouseMove, onSplitterMouseMove),
+    [props.onMouseMove, onSplitterMouseMove]
+  );
```

Comment:
Realized we were listener leaking due to re-renders occurring then event function references changing. As a result we would be unable to remove the listener on the disengagement of touch or mouse transaction. 😨